packages/react-native-aztec/README.md

```diff
@@ -0,0 +1,9 @@
+## Notice: We have git subtree'd this repo into https://github.com/wordpress-mobile/gutenberg-mobile. It is recommended that any new development happens on that one instead. Thanks!
```

Comment:
Is this README still valid?

Reply:
Updated the README to remove this line 👍